kubeflow-katib.advisories.yaml

```diff
@@ -22,6 +22,16 @@ advisories:
         data:
           fixed-version: 0.15.0-r7
 
+  - id: CVE-2023-44487
+    aliases:
+      - GHSA-m425-mq94-257g
```

Comment:
Are you sure this alias is correct? I see it reported on 2 different lines in `wolfictl scan`

Reply:
CVE-2023-44487 reports as a false-positive by `govulncheck` - we should record the advisory as such (even if we bumped the version of google.golang.org/grpc

Reply:
for the grpc issue this is what we did for the others as well